scalafix-core/src/main/scala/scalafix/util/Api.scala

```diff
@@ -16,6 +19,11 @@ trait Api {
     def asPatch: Patch = patch.getOrElse(Patch.empty)
   }
 
+  implicit class XtensionScalafixTreeInspect(tree: Tree) {
+    def inspect: String = Inspect.pretty(tree, showFieldNames = false)
+    def inspectLabeled: String = Inspect.pretty(tree, showFieldNames = true)
+  }
+
```

Comment:
I would also add an extension for `Option[Tree]` and `List[Tree]`.

Reply:
Sure thing, done.